.github/workflows/kfp-tekton-unittests.yml

```diff
@@ -109,3 +109,45 @@ jobs:
         uses: actions/checkout@v2
       - name: "run go pipelineLoop unit tests"
         run: cd tekton-catalog/pipeline-loops && make test-all
+  backend-integration:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: backend changes
+        id: backend-changes
+        run: |
+          git fetch
+          case "${{ github.event_name }}" in
+            pull_request)
+              echo "::set-output name=all::$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep -e '^backend/' | grep -v '^backend/.*README.md' | xargs)"
```

Comment:
can you also check the go.mod file?

Reply:
sure, see if I can find an action to filter change lists. that will be cleaner

Reply:
added `go.mod` now

Reply:
unfortunately, I can't find a good GitHub action to do change filtering. still need to code the filtering logic